libmozevent/phabricator.py

```diff
@@ -46,6 +46,7 @@ def __init__(self, request):
         self.reviewers = []
         self.diff = None
         self.stack = []
+        self.initial_base_revision = None
```

Comment:
You can simply name it  `base_revision`, that's the one on the build (not necesseraly the one we use to clone though).